tests/python_helics/test_value_federate.py

```diff
@@ -108,58 +97,49 @@ def test_value_federate_runFederateTestNamedPoint(vFed):
     testValue2 = "I am a string"
     testVal2 = 0.0
 
-    pubid = h.helicsFederateRegisterGlobalTypePublication(vFed, "pub1", h.HELICS_DATA_TYPE_NAMEDPOINT, "")
-    subid = h.helicsFederateRegisterSubscription(vFed, "pub1", "named_point", "")
+    pubid = h.helicsFederateRegisterGlobalPublication(vFed, "pub1", h.HELICS_DATA_TYPE_NAMEDPOINT, "")
+    subid = h.helicsFederateRegisterSubscription(vFed, "pub1", "")
 
-    status = h.helicsSubscriptionSetDefaultNamedPoint(subid, defaultValue, defVal)
-    assert status == 0
+    h.helicsInputSetDefaultNamedPoint(subid, defaultValue, defVal)
 
-    status = h.helicsFederateEnterExecutionMode(vFed)
-    assert status == 0
+    h.helicsFederateEnterExecutingMode(vFed)
 
     # publish string1 at time=0.0;
-    status = h.helicsPublicationPublishNamedPoint(pubid, testValue1, testVal1)
-    assert status == 0
+    h.helicsPublicationPublishNamedPoint(pubid, testValue1, testVal1)
 
     # double val;
-    status, value, val = h.helicsSubscriptionGetNamedPoint(subid)
-    assert status == 0
-    assert value == defaultValue
-    assert val == defVal
+    val = h.helicsInputGetNamedPoint(subid)
+    # assert value == defaultValue
+    assert val == ['start of a longer string in place of the shorter one and now this should be very long', 5.3]
```

Comment:
Should you use the named variable here, so you don't have to change the string in two places if you want a different string?  Or is there some reason this won't work

Reply:
I made this change.